docs/resources/job.md

```diff
@@ -357,7 +357,7 @@ This block describes the queue settings of the job:
 This block is used to specify Git repository information & branch/tag/commit that will be used to pull source code from to execute a job. Supported options are:
 
 * `url` - (Required) URL of the Git repository to use.
-* `provider` - (Optional, if it's possible to detect Git provider by host name) case insensitive name of the Git provider.  Following values are supported right now (could be a subject for change, consult [Repos API documentation](https://docs.databricks.com/dev-tools/api/latest/repos.html)): `gitHub`, `gitHubEnterprise`, `bitbucketCloud`, `bitbucketServer`, `azureDevOpsServices`, `gitLab`, `gitLabEnterpriseEdition`.
+* `provider` - (Required, if it's possible to detect Git provider by host name) case insensitive name of the Git provider.  Following values are supported right now (could be a subject for change, consult [Repos API documentation](https://docs.databricks.com/dev-tools/api/latest/repos.html)): `gitHub`, `gitHubEnterprise`, `bitbucketCloud`, `bitbucketServer`, `azureDevOpsServices`, `gitLab`, `gitLabEnterpriseEdition`.
```

Comment:
don't need this change - we try to detect `provider` based on host name

Reply:
Ah right, let me revert this part